packages/components/src/components/input/input.ts

```diff
@@ -279,6 +280,13 @@ export default class SdInput extends SolidElement implements SolidFormControl {
     event.stopPropagation();
   }
 
+  private handleSearchClick(event: MouseEvent) {
+    this.value = '';
+    this.emit('sd-search');
```

Comment:
Not sure if we should have sd-search-click, sd-search-blur etc. here. Please align with @paulovareiro29 and @smfonseca